### DISCUSSION TITLE
Unexpected token '<' SyntaxError: Unexpected token '<'. Other Fixes arn't working

### DISCUSSION THREAD

**Discussion by DaraghWallace** [jeffreylanters/react-unity-webgl#542] [Help Wanted!]
I have a very basic WEBGL build and i can't get past this error.
![ErrorCapture](https://github.com/jeffreylanters/react-unity-webgl/assets/121596764/2920c983-bd28-4bc5-b84e-67651f077646)

![CodeCapture](https://github.com/jeffreylanters/react-unity-webgl/assets/121596764/1b07045d-4770-4d1f-b340-173744056b99)

**Comment by amirali-bim** [comment]
facing same problem @jeffreylanters with react version 18.2.0 and Unity version 2021.3.28 LTS empty build for WEBGL.

**Comment by LukeWaterhouse** [comment]
I'm getting this too

**Comment by jeffreylanters** [comment]
You've incorrectly configures your project, please see the follow documentation: https://react-unity-webgl.dev/docs/main-concepts/unity-config
